src/components/OSCALCatalogGroup.js

```diff
@@ -130,13 +130,25 @@ function OSCALCatalogControlListItem(props) {
 
 function OSCALCatalogGroupList(props) {
   return (
-    <CollapseableListItem itemText={props.group.title}>
+    <CollapseableListItem
+      itemText={props.group.title}
+      controlId={props.control?.id}
+      urlFragment={props.urlFragment}
```

Comment:
I would love if we could begin destructuring more of these `props.___` things. At the very least, can we do that where we are using `urlFragment`?